docs/src/main/sphinx/object-storage/file-system-s3.md

```diff
@@ -262,6 +262,8 @@ Example JSON configuration:
   - The name of the *extra credential* used to provide the IAM role.
 * - `s3.security-mapping.kms-key-id-credential-name`
   - The name of the *extra credential* used to provide the KMS-managed key ID.
+* - `s3.security-mapping.customer-key-credential-name`
```

Comment:
Did you consider adding `sse` in the property name?

Reply:
I was following how the config was named for kms key id 
i.e `s3.sse.kms-key-id` is named as `s3.security-mapping.kms-key-id-credential-name` here, so I named `s3.sse.customer-key` as `s3.security-mapping.customer-key-credential-name` .
 I kinda agree that `customer-key` is not very self explanatory like `kms-key-id`.